appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java

```diff
@@ -150,15 +151,35 @@ public VmApiProxyDelegate() {
       int timeoutMs,
       boolean wasAsync) {
     // If this was caused by an async call we need to return the pending call semaphore.
+    long start = System.currentTimeMillis();
     environment.apiCallStarted(VmRuntimeUtils.MAX_USER_API_CALL_WAIT_MS, wasAsync);
     try {
-      return runSyncCall(environment, packageName, methodName, requestData, timeoutMs);
+      byte responseData[] = runSyncCall(environment, packageName, methodName, requestData, timeoutMs);
+      long end = System.currentTimeMillis();
+      logger.log(Level.INFO, String.format(
+              "Service bridge API call to package: %s, call: %s, of size: %s " +
+              "complete. Service bridge status code: %s; response " +
+              "content-length: %s. Took %s ms.",
+              packageName, methodName, requestData.length, 200,
+              responseData.length, (end - start)));
```

Comment:
Do you have a link in the console for such a log content? That would help as well.